packages/calcite-components/src/components/stepper/stepper.tsx

```diff
@@ -258,6 +277,20 @@ export class Stepper {
   /** keep track of the currently active item position */
   private currentPosition: number;
 
+  private mutationObserver = createObserver("mutation", () => this.updateItems());
+
+  private updateItems(): void {
+    if (this.items) {
```

Comment:
Shouldn't it be querying the items here instead of relying on the register event? Otherwise, watching this.items is ncessary to updateItems().